### PR TITLE
Ep 13215 modularization

### PR DESCRIPTION
**Pull request by A-Gochnio**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,2 +1 @@
-export * from './src/app/common';
-export * from './src/app/dashboards';
+export const IMPORT_SUBMODULE_INSTEAD = 'import submodule instead';
```
Comment (bwitkowicz) [inline index.ts]: what does it do?

File: src/app/add-input/add-input.component.html [hunk 2]
```diff
@@ -2,4 +2,4 @@
        [placeholder]="placeholder"
        [(ngModel)]="value"
        (keydown.enter)="submit()">
-<button [disabled]="!value.length" (click)="submit()">Add</button>
+<button [disabled]="!value.length" (click)="submit()" translate>Add</button>
```
Comment (bwitkowicz) [inline src/app/add-input/add-input.component.html]: Are you sure the `Add` key will get picked up by the parser?

File: src/app/add-input/add-input.component.sandbox.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { sandboxOf } from 'angular-playground';
+import { AddInputComponent } from './add-input.component';
+import { FormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+
```
Comment (bwitkowicz) [inline src/app/add-input/add-input.component.sandbox.ts]: nl

File: src/app/paginator/paginator.component.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
-@import '../../../../node_modules/@enplug/dashboard-sdk-utils/src/sass/global/_brand';
-@import '../../../../node_modules/@enplug/dashboard-sdk-utils/src/sass/global/_brand-vars';
+@import '../../../node_modules/@enplug/dashboard-sdk-utils/src/sass/global/_brand';
+@import '../../../node_modules/@enplug/dashboard-sdk-utils/src/sass/global/_brand-vars';
```
Comment (bwitkowicz) [inline src/app/paginator/paginator.component.scss]: quite ugly import, but I guess there's nothing we can do about that ;)

File: index.ts (hunk 1 above)
Comment (A-Gochnio) [reply index.ts]: When a user tries to import something from `@enplug/components` <-- the root of the package, his IDE will autocomplete `IMPORT_SUBMODULE_INSTEAD`, which is a hint that nothing resides in the root of the package.

File: src/app/add-input/add-input.component.html (hunk 2 above)
Comment (A-Gochnio) [reply src/app/add-input/add-input.component.html]: I am not

File: src/app/paginator/paginator.component.scss (hunk 4 above)
Comment (A-Gochnio) [reply src/app/paginator/paginator.component.scss]: It is.
The author of the component can and should do something about it, we should not rely on `@dashboard-sdk-utils` anywhere, as it is supposed to be deprecated.

Comment (seishin4real) [reply src/app/paginator/paginator.component.scss]: yeah, I've got a card for that - just need to find time ;)

File: src/app/click-outside/click-outside.module.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { ClickOutsideDirective } from './click-outside.directive';
+
+@NgModule({
+  imports: [],
```
[the remark this answers is not in the document]
Comment (A-Gochnio) [reply src/app/click-outside/click-outside.module.ts]: Yep, no dependencies here.

File: src/app/click-outside/click-outside.module.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+  declarations: [ ClickOutsideDirective ],
+  providers: [],
```
[the remark this answers is not in the document]
Comment (A-Gochnio) [reply src/app/click-outside/click-outside.module.ts]: Yep, no providers in this module.

**Comment by A-Gochnio** [comment]
Code review time out :) I'll merge now.

**Pull request merged**